suitcase/pizza_box/handlers_legacy.py

```diff
@@ -0,0 +1,46 @@
+from collections import namedtuple
+
```

Comment:
We should add some quick text explaining what this is:
```py3
'''
These are legacy file handlers first implemented at ISS. To the
knowledge of the authors, they were never implemented in production.
They work differently (and less efficiently) from the accepted handlers
and their use is strongly discouraged. Since it is not easy to fully
capture the extent to which these handlers could have been used, they
are left here in the event they need to be referred to.
'''

```